TCC/TccController.cs

```diff
@@ -61,6 +70,71 @@ public async Task BenchmarkAsync(BenchmarkOption option)
         await LogResultAsync(operationResult, Mode.Benchmark, null);
     }
 
+    public async Task AutoDecompressAsync(AutoDecompressOptionBinding option)
+    {
+        var gcpCredential = await GoogleAuthHelper.GetGoogleClientAsync(option.GoogleStorageCredential, new CancellationToken());
+        var subscriber = await GetGoogleClientAsync(gcpCredential, option);
+        var storage = await StorageClient.CreateAsync(gcpCredential);
+
+        // Connect et on appelle DecompressAsync
+        await _databaseSetup.EnsureDatabaseExistsAsync(Mode.Decompress);
+
+        // Use the client as you'd normally do, to listen for messages in this example.
+        await subscriber.StartAsync(async (msg, cancellationToken) =>
```

Comment:
Is this call blocking ? (aka : tcc is still running, waiting for new events ?)
If this is the case : how does TCC must be preferably launched ? as a windows service ?

Reply:
Yes it's blocking, waiting for new events.

You can start it with [nssm](https://nssm.cc/usage) or as a scheduled task.